Source/ee/Ee_SubSystem.h

```diff
@@ -52,8 +55,8 @@ namespace Ee
 		CDMAC						m_dmac;
 		CGIF						m_gif;
 		CSIF						m_sif;
-		CVpu						m_vpu0;
-		CVpu						m_vpu1;
+		CVpu*						m_vpu0;
```

Comment:
Considering the previous lifetime of this object (tied to the parent as an inline member) `unique_ptr` would be the direct 1:1 semantic replacement.